account-integrations/safe/package.json

```diff
@@ -33,7 +33,6 @@
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-prettier": "^5.0.0",
     "ethers": "^6.4.0",
-    "ethers-v5": "npm:ethers@5.7.2",
```

Comment:
When I originally copied over these tests and converted them to use ethers v6, I ran into an issue where the bundlerProvider would not work if it wasn't connected to an ethers v5 provider. Hence I added this alias. Did you run into this issue at all? It looks like you just started using ethers v6 for the bundlerProvider with no issues

Reply:
Oh interesting. Yeah I didn't run into any trouble, wonder what happened.